frontend/network/src/main/java/com/intersoft/network/models/responses/EventBody.kt

```diff
@@ -15,3 +15,15 @@ data class EventData(
     val location : String,
     val owner_id : Int
 )
+
+data class StoredEventData(
```

Comment:
In the end, I don't see the usage of the StoredEventData since it stores less information than the NewEventData (holds only the names of the participants instead of the whole class), and the NewEventData can be used in more cases like the event creation body. The only advantage to me of the StoredEventData is that it allows for easier storage of participants' names, but because email is the primary key, having stored only the names of users makes it harder for when I want to confirm if the authenticated user is a participant or not.

Reply:
The "StoredEventData" data class is created based on the backend documentation that specifies what the server returns upon a get request for a specific event id. The server does not return an email so there is no point in having it in the data class.

Reply:
Okay, part of me understands what happened. I had an EventData. And when I merged to yours, it mixed the NewEventData with the EventData, making NewEventData and StoredEventData equal. What should be changed is the participants list being a list of String and not a list of UserData like I have in my PR.

Reply:
The participants list IS a List of strings 
![image](https://github.com/DMindek/GroupUP/assets/100722922/68c99b3f-22b3-4d65-8bfc-b6ab6f9bdd06)

Am I misunderstanding what the requested change is?

Reply:
Sorry, maybe I misspoke. It should be a list of UserData instead of String so that we can get information like userId and username from the object.